Client/Frontend/Autofill/CreditCard/CreditCardSettingsView/CreditCardSettingsViewController.swift

```diff
@@ -150,4 +149,19 @@ class CreditCardSettingsViewController: UIViewController, Themeable {
     deinit {
         notificationCenter.removeObserver(self)
     }
+
+    // MARK: - Private helpers
+
+    private func updateStateForEditView() {
+        if AppAuthenticator.canAuthenticateDeviceOwner() {
```

Comment:
I think we should use a `guard` for this instead of an `if`.

Reply:
We should probably also inject `AppAuthenticator` so the class is testable. Ideally `AppAuthenticator` should get a protocol as well so we can mock it in tests.